### PR TITLE
create sameAs relationship for detroit city, MI wards

### PR DESCRIPTION
**Pull request by nuttingd1**
In the current [Detroit charter](https://detroitmi.gov/sites/detroitmi.localhost/files/2018-05/2_29_2012_CharterDocument_2_1_WITHOUT_COMMENTARY_1.pdf), 'districts' and 'wards' are used interchangeably to describe the drawn political units for which the city council, board of police commissioners, and community advisory councils are elected (§2-105.A.12 and §3-108), and I think they should be aliased in the repository. I'd like to make the `council_district` types the preferred identifier, since the city council refers to their political geographies as 'districts' on their [official website](https://detroitmi.gov/government/city-council), [interactive map](https://detroitmi.gov/webapp/interactive-district-map), and [census data map](https://detroitmi.gov/webapp/census-data-map) ('districts' are similarly used on the [board of police commissioners](https://detroitmi.gov/government/boards/board-police-commissioners) and [community advisory council](https://detroitmi.gov/government/city-council/city-council-district-4/district-4-community-advisory-council) websites).

Additionally, there are only 7 electoral districts from which members of these bodies are elected (save for the at-large members), so I added validThrough dates for wards 8-22 currently in the repository for this year. It doesn't look like there have been more than 7 electoral districts in the recent past, and I can't find any instances where these additional districts are currently in use. Not sure what the right move is here, but I could see removing them from the repository altogether if that makes more sense.

TLDR, here are the changes I made:
-create mi-aliases file
-move 'ward' identifiers to mi-aliases file, add validThrough dates for wards no longer in use
-make 'council_district' the preferred identifier by adding sameAs fields for wards 1-7

### CODE REVIEW

**Comment by nuttingd1** [comment]
@jloutsenhizer @jpmckinney just tagging y'all for review, thanks!

**Comment by jpmckinney** [comment]
@nuttingd1 Can you investigate why the diff for `identifiers/country-us/state-mi-local_gov.csv` is the entire file? Did the line endings change? Preferably, the commit would be substituted for one that produces a smaller diff.

Also, this branch conflicts, so that needs to be resolved.

**Comment by nuttingd1** [comment]
submitted a new pull request to address the diff issues in #356
